core/src/components/picker-column/picker-column.tsx

```diff
@@ -81,14 +71,23 @@ export class PickerColumnCmp implements ComponentInterface {
       onEnd: ev => this.onEnd(ev),
     });
     this.gesture.setDisabled(false);
-
     this.tmrId = setTimeout(() => {
       this.noAnimate = false;
       this.refresh(true);
     }, 250);
   }
 
-  componentDidUnload() {
+  componentDidLoad() {
+    // get the height of one option
```

Comment:
add comment of why this needs t be here:
// It needs to perform a DOM read after the first render